patches/expose-flags.patch

```diff
@@ -1,38 +1,38 @@
 diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
-index 96370d4ac35a9..bc80527377261 100644
+index 96370d4ac35a9..6429e45f5013e 100644
 --- a/chrome/browser/about_flags.cc
 +++ b/chrome/browser/about_flags.cc
 @@ -4190,6 +4190,36 @@ const FeatureEntry kFeatureEntries[] = {
  // //tools/flags/generate_unexpire_flags.py.
  #include "build/chromeos_buildflags.h"
  #include "chrome/browser/unexpire_flags_gen.inc"
-+    {"incognito-launch",
-+     "Incognito Launch",
++    {"incognito-launch", "Incognito Launch",
 +     "Launch external links and open new sessions in Incognito. Disabled by "
 +     "default. This flag is provided by hardened-chromium.",
 +     kOsAll, FEATURE_VALUE_TYPE(features::kIncognitoLaunch)},
-+    {"extensions-support",
-+     "Extensions Support",
-+     "Toggle extensions support. This flag is exposed by hardened-chromium.",
-+     kOsAll, SINGLE_DISABLE_VALUE_TYPE("disable-extensions")},
-+    {"disable-cross-origin-referrers",
-+     "Clear cross-origin referrers",
++    {"extensions-support", "Extensions Support",
++     "Toggle extensions support. This switch is exposed by hardened-chromium.",
++     kOsAll, SINGLE_DISABLE_VALUE_TYPE(switches::kDisableExtensions)},
++    {"v8-jit-support", "Webassembly and JavaScript JIT",
```

Comment:
`global-disable-v8-jit-support`?

Reply:
Oh I see what you mean, yeah it's just also potentially confusing in that if someone comes across the toggle, they may see v8-jit-support: enabled by default and wonder if that means jit is on by default. Whereas what this toggle is in practice doing is providing a global override to disable jit for all pages. It doesn't really ever enable v8-jit-support globally.

Reply:
whereas `force-disable-v8-jit: disabled (default)` seems more descriptive of the actual behavior of the toggle

Reply:
even though it results in an inverted boolean